Dockerfile

```
@@ -97,5 +100,6 @@ RUN \
 
 # add local files
 COPY root/ /
+COPY ci-check/ /package/admin/s6-overlay-${S6_OVERLAY_VERSION}/etc/s6-rc/sources/top/contents.d
```

Move this up around like 50 (where the docker mod stuff is patched in) and change it to:
```suggestion
COPY ci-check/ /root-out/package/admin/s6-overlay-${S6_OVERLAY_VERSION}/etc/s6-rc/sources/top/contents.d
```